### PR TITLE
VBM-140: incorrect time format and gray background fixed

### PR DESCRIPTION
**Pull request by mazurGit**

### CODE REVIEW

File: packages/mobile/src/common/enums/product/product-status.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+enum ProductStatus {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/common/enums/product/product-status.ts:1]: I believe we should have that enum in `shared`

File: packages/mobile/src/screens/home/components/product/product.tsx [hunk 2]
```diff
@@ -40,69 +50,80 @@ const Product: FC<Props> = ({ productId }) => {
...
+          <View style={styles.imgWrapper}>
+            <Image source={{ uri: imageLinks[0] }} style={styles.img} />
+
+            {isAuction && (
+              <TimeWindow
+                duration={timeToAuctionEnd}
+                style={{
+                  position: 'absolute',
+                  bottom: -10,
+                  alignSelf: 'center',
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/home/components/product/product.tsx:68]: Place it into `styles` object

File: packages/shared/src/common/types/product/product-dto.ts [hunk 3]
```diff
@@ -32,6 +32,7 @@ type ProductDto = {
   currentPrice: number;
   author: Author;
   category: Category;
+  createdAt: Date;
```
Comment (AleksandrTermenzhy) [inline packages/shared/src/common/types/product/product-dto.ts:35]:
```suggestion
  createdAt: string;
```

File: packages/mobile/src/common/enums/product/product-status.ts (hunk 1 above)
Comment (mazurGit) [reply packages/mobile/src/common/enums/product/product-status.ts:1]: Doesn't exist

File: packages/frontend/components/product/timer/types.ts [hunk 4]
```diff
@@ -1,3 +1,3 @@
 export interface TimerProps {
-  date: Date;
+  date: string;
```
Comment (AleksandrTermenzhy) [inline packages/frontend/components/product/timer/types.ts:2]: Won't it broke something on the frontend?

File: packages/shared/src/common/types/product/product-dto.ts [hunk 5]
```diff
@@ -26,12 +26,13 @@ type ProductDto = {
   type: ProductType;
   status: string;
```
Comment (AleksandrTermenzhy) [inline packages/shared/src/common/types/product/product-dto.ts:27]:
```suggestion
  status: ProductStatus;
```

**Review by AleksandrTermenzhy — APPROVED** [review]

**Review by dolphin323 — APPROVED** [review]

**Review by natata7 — APPROVED** [review]

**Pull request merged**